contracts/triggers/AaveQuotePriceTrigger.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+import "../interfaces/ITrigger.sol";
+import "../interfaces/aaveV3/IAaveV3Oracle.sol";
+import "../interfaces/lido/IWStEth.sol";
+import "../auth/AdminAuth.sol";
+import "../DS/DSMath.sol";
+import "../utils/TokenUtils.sol";
+import "../actions/aaveV3/helpers/AaveV3RatioHelper.sol";
+
+/// @title Trigger contract that verifies if current token price ratio is over/under the price ratio specified during subscription
+contract AaveQuotePriceTrigger is ITrigger, AdminAuth, DSMath, AaveV3RatioHelper {
```

Comment:
Would rename to AaveV3QuotePriceTrigger to avoid confusion as we have a V2 and V3 ratio trigger as well, same for Trailing